app/scripts/controllers/network/provider-api-tests/shared-tests.js

```diff
@@ -572,9 +572,31 @@ export function testsForRpcMethodAssumingNoBlockParam(
     });
   });
 
-  it.each([null, undefined, '\u003cnil\u003e'])(
-    'does not reuse the result of a previous request if it was `%s`',
-    async (emptyValue) => {
+  for (const emptyValue of [null, undefined, '\u003cnil\u003e']) {
```

Comment:
this will make it more difficult to reflect these changes onto https://github.com/MetaMask/core/pull/1087.

Please limit the diff on these tests to changes that are absolutely necessary.